frontend/src/metabase/components/SingleSelectListField/types.ts

```diff
@@ -7,7 +7,9 @@ export interface SingleSelectListFieldProps {
   value: RowValue[];
   options: Option;
   optionRenderer: (option: any) => JSX.Element;
+  onSearchChange: (prefix: string) => void;
```

Comment:
nit: this does not have to be a prefix

```suggestion
  onSearchChange: (value: string) => void;
```